src/ucs/sys/topo/base/topo.c

```diff
@@ -334,17 +334,7 @@ static int ucs_topo_is_pci_root(const char *path)
 
 static void ucs_topo_sys_root_distance(ucs_sys_dev_distance_t *distance)
 {
-    distance->latency = 500e-9;
-    switch (ucs_arch_get_cpu_model()) {
-    case UCS_CPU_MODEL_AMD_ROME:
-    case UCS_CPU_MODEL_AMD_MILAN:
-    case UCS_CPU_MODEL_AMD_GENOA:
-        distance->bandwidth = 5100 * UCS_MBYTE;
-        break;
-    default:
-        distance->bandwidth = 220 * UCS_MBYTE;
-        break;
-    }
+    *distance = ucs_global_opts.sys_root;
```

Comment:
Minor: this function is called only once and does only one action so maybe it worth to just make assignment in place of function call and remove this function?

Same for `ucs_topo_common_numa_node_distance`

Reply:
Removed.